app/src/main/java/com/rentminder/MainActivity.kt

```diff
@@ -55,34 +56,20 @@ class MainActivity : ComponentActivity() {
 
 @Composable
 fun MainMenu() {
-    //Getting current month for Main Menu
-    val cal: Calendar = Calendar.getInstance()
-    val monthDate = SimpleDateFormat("MMMM")
-    val monthName: String = monthDate.format(cal.time)
 
-    Column() {
-        TopToolBar()
-        Row(modifier = Modifier
-            .padding(
-                top = 5.dp,
-                bottom = 10.dp
-            )
-            .fillMaxWidth(),
-            horizontalArrangement = Arrangement.Center
-        ) {
-            Text(
-                text = monthName,
-                fontSize = 28.sp,
-                fontWeight = FontWeight.Bold
-            )
-        }
-        EditBillAmounts()
+    Column {
+        TopNavBarColumn()
+        CalendarNameColumn()
+        RentBillColumn()
     }
 }
 
-//Navigation bar at the top of app
+/**
+ * 1. Column (TopNav)
+ * Creates navigation bar at the top of the app.
+ */
```

Comment:
A good programming practice is to provide good documentation for other developers so that they may understand how to efficiently navigate the programs that you provide.

To follow proper Kdoc standards, always put a /** at the beginning of all classes and functions. From a front perspective, I always find it critical to describe the layout. In this case, I use the implementations of columns and rows to describe how the UI is being presented to the user.

Reply:
Yes, proper KDoc format is preferred, over functions.